Trading/Exchange/binance_websocket_feed/metadata.json

```diff
@@ -0,0 +1,6 @@
+{
+  "version": "1.2.0",
+  "origin_package": "OctoBot-Default-Tentacles",
+  "tentacles": ["BinanceWebsocketFeed"],
```

Comment:
should be the same as the name of the tentacle class (`BinanceCryptofeedWebsocketConnector`)